### DISCUSSION TITLE
请求返回 1034 该怎么解决呀

### DISCUSSION THREAD

**Discussion by 6wz** [UIGF-org/mihoyo-api-collect#36] [Q&A]
请求CURL  ： 

curl --location --request GET 'https://api-takumi-record.mihoyo.com/game_record/app/genshin/api/index?server=cn_gf01&role_id=298004554' \
--header 'x-rpc-client_type: 5' \
--header 'x-rpc-app_version: 2.63.1' \
--header 'User-Agent: apifox/1.0.0 (https://www.apifox.cn)' \
--header 'Cookie: ltoken={LTOKEN}; ltuid=409509855'

**Comment by BTMuli** [comment]
[错误码参照](https://github.com/UIGF-org/mihoyo-api-collect/blob/main/other%2Ferror_code.md)
